### ISSUE TITLE
Make it possible to list existing factoids

### ISSUE THREAD

**Issue by AlexDaniel** [zoffixznet/perl6-IRC-Client-Plugin-Factoid#3]
Perhaps `^list` or something.

**Comment by zoffixznet** [comment]
Yes, that's planned. The final version will have [search](https://github.com/zoffixznet/perl6-IRC-Client-Plugin-Factoid/blob/master/DESIGN.md#search) and [dumping all factoids](https://github.com/zoffixznet/perl6-IRC-Client-Plugin-Factoid/blob/master/DESIGN.md#level-2-implementation-dump-all).
